fableous-fe/src/containers/HubCanvasPage.tsx

```diff
@@ -26,49 +52,96 @@ export default function HubCanvasPage() {
           default:
         }
       } catch (e) {
-        console.log(e);
+        console.error(e);
       }
     };
-    const ping = setInterval(
+    const interval = setInterval(
```

Comment:
is this ping interval needed? theres one above on wsRef.current.onopen already

Reply:
done